doc/ws.md

```diff
@@ -139,7 +139,11 @@ returned as a string.
 
 ### server.close([callback])
 
-Close the server and terminate all clients, calls callback when done.
+Close the server and terminate all clients, calls callback when done. The
```

Comment:
Can you please reword like this?

```
Close the HTTP server if created internally, terminate all clients and call
callback when done. If an external HTTP server is used via the `server` or
`noServer` constructor options, it must be closed manually.
```

I think it's more accurate.

Thank you.

Reply:
Okay, I have updated it.